libs/event/src/lib/form/shell/shell.component.ts

```diff
@@ -61,7 +65,26 @@ export class EventFormShellComponent implements OnInit, OnDestroy {
     // @TODO #6780
     this.sub = eventId$.pipe(
       switchMap((eventId: string) => this.eventService.valueChanges(eventId))
-    ).subscribe(event => {
+    ).subscribe(async event => {
+
+      const { eventId } = this.route.snapshot.params;
+      const params = this.route.snapshot.queryParams;
+      if (eventId === 'new' && params) {
+        const { titleId } = params;
```

Comment:
I think queryParams is [always an object](https://angular.io/api/router/ActivatedRouteSnapshot#queryParams) so you can safely do :
```suggestion
      const { titleId } = this.route.snapshot.queryParams;
      if (eventId === 'new' && titleId) {
```